pkgs/os-specific/linux/sysdig/default.nix

```diff
@@ -72,6 +72,13 @@ stdenv.mkDerivation rec {
 
     cp -r ${driver} driver-src
     chmod -R +w driver-src
+
+    # Hacky but needed until https://github.com/draios/sysdig/issues/2077 is resolved for kernel >= 6.8 as strlcpy got removed and build fails
```

Comment:
This is hacky but the simplest solution (I found) to fix the build and due to the condition in the code I'm sure it's not executed for 6.8-kernel - is there a better way?

Tried combinations with "-Wno-error" etc. but seems to not be help.

Reply:
The issue is this is a build error breaking the build so I can't ignore it without patching it...